validation/src/test/scala/hmda/validation/rules/lar/validity/PurchaserTypeUtils.scala

```diff
@@ -0,0 +1,16 @@
+package hmda.validation.rules.lar.validity
```

Comment:
I put it there (and with that name) because at the moment, that's the only place where it's used, and the only kind of thing it contains. I would definitely be open to generalizing it and/or putting it somewhere more obvious, though.